content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -106,6 +106,10 @@ reproduces the same set of steps that the kubelet performs to calculate
 file-backed memory on the inactive LRU list) from its calculation, as it assumes that
 memory is reclaimable under pressure.
 
+On Windows nodes, the value for `memory.available` is derived from the node's global
+memory commit levels (queried through the [GetProformanceInfo](https://learn.microsoft.com/windows/win32/api/psapi/nf-psapi-getperformanceinfo)
+system call) by subtracting the node's global [`CommitTotal`](https://learn.microsoft.com/windows/win32/api/psapi/ns-psapi-performance_information) from the node's [`CommitLimit`](https://learn.microsoft.com/windows/win32/api/psapi/ns-psapi-performance_information). Please note that `CommitLimit` can change if the node's page-file size changes!
+
```

Comment:
Because there's now more content, I recommend:
```suggestion

### Filesystem layout 

The way that you lay out filesystems on a node is relevant to the eviction signals that the kubelet
might emit.
```

Reply:
Thanks - should I also add a L3 heading for memory management?
I was debating doing so.

Reply:
Feel free!